src/Makefile.am

```diff
@@ -19,6 +19,12 @@ librfxencode_la_LIBADD = x86/librfxencode-x86.la sse2/librfxencode-sse2.la
 AM_CPPFLAGS += -Isse2 -DSIMD_USE_ACCEL=1 -DRFX_USE_ACCEL_X86=1
 endif
 
+if WITH_SIMD_ARM64
+SUBDIRS += neon
+librfxencode_la_LIBADD = neon/librfxencode-neon.la
+AM_CPPFLAGS += -Ineon -DSIMD_USE_ACCEL=1 -DRFX_USE_ACCEL_ARM64=1
```

Comment:
I can't test this as I don't have suitable hardware, but I'd assume you need `-I$(top_srcdir)/src/neon` here (see #67)

Reply:
Thanks Matt, look like I missed that again.